library/sbi-rt/src/base.rs

```diff
@@ -146,6 +146,10 @@ define_extension! {
     Cppc(sbi_spec::cppc::EID_CPPC) /// SBI CPPC extension.
     Nacl(sbi_spec::nacl::EID_NACL) /// Nested Acceleration extension.
     Sta(sbi_spec::sta::EID_STA) /// Steal-time Accounting extension.
+    Sse(sbi_spec::sse::EID_SSE) /// Supervisor Software Events extension.
+    Fwft(sbi_spec::fwft::EID_FWFT) /// Firmware Feature Tracking extension.
+    Dbtr(sbi_spec::dbtr::EID_DBTR) /// Debug Trigger extension.
+    Mpxy(sbi_spec::mpxy::EID_MPXY) /// Multi-plexing extension.
```

Comment:
```suggestion
    Mpxy(sbi_spec::mpxy::EID_MPXY) /// Message Proxy extension.
```
Please read the RISC-V SBI specification carefully. Link: [here](https://github.com/riscv-non-isa/riscv-sbi-doc/blob/1f3c7c01f03f15f02f7de036663e0d0865a550e1/src/ext-mpxy.adoc#message-proxy-extension-eid-0x4d505859-mpxy)

Reply:
OK, thank you, my bad